front/src/applications/operationalStudies/views/SimulationResults.tsx

```diff
@@ -65,6 +66,13 @@ const SimulationResults = ({
     infraId
   );
 
+  const pathItemsCoordinates =
+    path && path.length && path.path_item_positions
+      ? path.path_item_positions.map((positionOnPath) =>
+          getPointCoordinates(pathProperties!.geometry, path.length, positionOnPath)
+        )
+      : [];
+
```

Comment:
Not sure why but sometimes the `pathItemsCoordinates` received by `RenderItinerary` are undefined even though you set them to an empty array by default here. So better to allow this const to be undefined for consistency. 

You also need pathProperties to the condition in case it's still computing (and remove the `!` from `getPointCoordinates`) 

```suggestion
  const pathItemsCoordinates =
    path && pathProperties
      && path.path_item_positions.map((positionOnPath) =>
          getPointCoordinates(pathProperties.geometry, path.length, positionOnPath)
        );

```

Reply:
done